evap/staff/views.py

```diff
@@ -1491,7 +1491,13 @@ def evaluation_textanswers(request, semester_id, evaluation_id):
         request.session["review-visited"] = visited
 
         sections = evaluation_sections + contributor_sections
-        template_data.update(dict(sections=sections, next_evaluations=next_evaluations))
+        # Evaluations where the grading process is finished should be shown first, need to be sorted in Python
+        template_data.update(
+            dict(
+                sections=sections,
+                next_evaluations=sorted(next_evaluations, key=lambda e: e.grading_process_is_finished, reverse=True),
```

Comment:
is sorting here necessary? aren't the evaluations already sorted correctly when being returned from `find_unreviewed_evaluations`?

Reply:
This is a leftover from https://github.com/e-valuation/EvaP/pull/1668/commits/d88c2ecc1ef4bc0f55ca6279c96f6ad24905b1d9. As discussed above in https://github.com/e-valuation/EvaP/pull/1668#pullrequestreview-808846464, it was replaced with the code in https://github.com/e-valuation/EvaP/pull/1668/commits/e50d2ac23687636b1d7f8d81af0f3ec062a333c5.

So, @PonderKoKo 

> I hope I did this correctly now.

Nope, not yet, but we are almost there: You want to remove those old changes from your history. One simple way to do that for now would be:
```bash
git status  # make sure you're on your reviewpriority branch, HEAD should be at 4506f94
git revert d88c2ec  # Add another commit that reverts the changes that we no longer want
git push origin
```

(Note: This keeps the original commit, and simply adds another one that undoes the changes. However, this is fine for now in the scope of this PR. If you're interested on how to "clean up" your commit history, you can take a look at git rebase, with its -i / --interactive option. Personal recommendation: Make a backup-branch before experimenting with that.)